src/components/auth/UserMenu.jsx

```
@@ -0,0 +1,203 @@
+import React, { useState } from 'react';
+import {
+  Button,
```

ESLint

[error] 'Button' is defined but never used.

(no-unused-vars)